INSTALL.md

```diff
@@ -1,3 +1,32 @@
-# Installing the ODF-CTD-PROC CTD Processing Utilities
+# ctdcal
 
-Yup... still need to work on this.
+This is an initial draft of the ctdcal installation process in its current form.
+
+### Clone repository
+Pull down the latest version of ctdcal:
+```
+git clone https://github.com/cchdo/ctdcal.git
+```
+
+### Install ctdcal and dependencies
+Change directories to the top-level ctdcal:
+```
+cd ctdcal
+```
+
+Create a new virtual environment with your preferred environment manager and run the setup script:
+```
+python setup.py [install/develop]
```

Comment:
I don't think I've personally run a `python setup.py` script directly in a few years now. It's always either: 
* `pip install .` or 
* `pip install -e .` (for dev mode)

Is this the "conda" way of doing stuff?

Reply:
I think I'm just out of touch with current ways of doing things

Reply:
Now I'm running into an issue where contents of `ctdcal/scripts/` are only accessible if installed in dev mode.
I guess we are on the way to moving from calling individual scripts to CLI, maybe I'll just do that instead

Reply:
Path issues have been resolved